loadgen/types.js

```diff
@@ -20,8 +20,6 @@ export {};
 /** @typedef {ReturnType<typeof import('@agoric/spawner').makeSpawner>} Spawner */
 /** @typedef {WalletUser & { getAdminFacet: () => WalletAdminFacet }} HomeWallet */
 
-/** @typedef { import('@agoric/run-protocol/src/vaultFactory/vaultFactory').VaultFactoryContract['publicFacet']} VaultFactoryPublicFacet */
-/** @typedef { import('@agoric/run-protocol/src/vaultFactory/vaultManager').CollateralManager} VaultCollateralManager */
```

Comment:
Why remove this? I'd like to avoid relying on ambient types.

Reply:
> avoid relying on ambient types.

Agreed.

> Why remove this?

Deep imports are worse than ambient types. Moreover these paths are broken.

Reply:
Removed